### ISSUE TITLE
Unable to build on Android  

### ISSUE THREAD

**Issue by b00f** [pactus-project/pactus#1506]
By upgrading Pactus to Go 1.23.x, we are no longer able to build Pactus for Android. The error we encounter is:

```
undefined: anet.Interfaces
```

After searching the internet, I found this issue mentioned here: https://github.com/wlynxg/anet/blob/main/README.md#how-to-build-with-go-1230-or-later. Unfortunately, this solution didn’t work, and I still received the same error.

To build Pactus for Android, you can run this command:

```
CGO_ENABLED=0 GOOS=android GOARCH=arm64 go build -ldflags "-s -w" -trimpath -o ./build/android/pactus-daemon ./cmd/daemon
```

**Comment by Ja7ad** [comment]
Currently `anet` package fixed issue on v0.0.5: https://github.com/wlynxg/anet/issues/7

**Comment by Ja7ad** [comment]
Parent package upgraded it to latest version: https://github.com/pion/transport/blob/master/go.mod

**Comment by Ja7ad** [comment]
Issue is tracking for fix in: https://github.com/libp2p/go-libp2p/issues/2997

**Comment by Ja7ad** [comment]
@FishDontExist Libp2p fixed it v0.37.0 please update dependencies:

https://github.com/libp2p/go-libp2p/releases/tag/v0.37.0

Please follow https://github.com/pactus-project/pactus/blob/main/docs/update-dependensies.md for update deps.
